spec/datetimeformat.html

```diff
@@ -360,6 +360,27 @@
         It is recommended that implementations use the time zone information of the IANA Time Zone Database.
       </emu-note>
     </emu-clause>
+
+    <emu-clause id="sec-unwrapdatetimeformat" aoid="UnwrapDateTimeFormat">
+      <h1>UnwrapDateTimeFormat( dtf )</h1>
+      <p>
+        The UnwrapDateTimeFormat abstract operation gets the underlying DateTimeFormat operation
+        for various methods which implement ECMA-402 v1 semantics for supporting initializing
+        existing Intl objects.
+      </p>
+      <emu-normative-optional><span class="normative-optional">Normative Optional</span><div class="normative-optional-contents">
+      <emu-alg>
+        1. If _dtf_ does not have an [[initializedDateTimeFormat]] internal slot and ? InstanceofOperator(_dtf_, %DateTimeFormat%),
```

Comment:
in general I haven't seen the use of ? in conditions... but I suspect that's ok.